### ISSUE TITLE
OSE development version does not allow upload of images

### ISSUE THREAD

**Issue by ppk4** [wireload/screenly-ose#568]
Today's development version does not allow me to upload images. 
Note: The dialog screen is different than in the 'normal' version.

Reverting back to the previous version worked fine.

**Comment by antonmolodykh** [comment]
you're right, the dialog screen now looks different.
what exactly happens when you try to upload images?

**Comment by ppk4** [comment]
Drag-and-drop does not work.  It opens the image in my browser. Dialog disappears.
Upload image does not work. No dialog. It doesn't allow me to attach a file.

**Comment by vpetersson** [comment]
@ppk4 what browser/version/os?

**Comment by ppk4** [comment]
Windows 7 SP1 and Firefox 53.0.2 (64-bit)

**Comment by antonmolodykh** [comment]
@ppk4 Today I've tried to use win 7, firefox 53.0.2, 64-bit. I haven't managed to reproduce the problem

**Comment by ppk4** [comment]
After the previous meesage, I switched back to the official release (not the development version).

Today, I reinstalled the development version. To my surprise the upload of image worked.

![screen shot 05-22-17 at 08 17 pm](https://cloud.githubusercontent.com/assets/28754534/26323983/31703c00-3f31-11e7-82f2-64808f715294.PNG)
![screen shot 05-22-17 at 08 18 pm](https://cloud.githubusercontent.com/assets/28754534/26323986/33164b4e-3f31-11e7-8e20-27c84bfec2f8.PNG)

According to the dialog screens, you do not know which image has been uploaded. You only get a message 'Loading is complete'.  However it worked.

If you want I can still upload the four files,
